### PR TITLE
Adding Feature Flag for Listening Mode

### PR DESCRIPTION
**Pull request by jsmorgan42**
# Description of Work
This PR adds in a feature flag that by default turns off the listening mode feature. However, if the feature needs to be turned on, a toggle has been added to the listening mode debug menu (accessed by quadruple tapping the text view at the top in the main screen). 

---

### CODE REVIEW

File: Vocable/AppConfig.swift [hunk 1]
```diff
@@ -20,22 +20,22 @@ struct AppConfig {
         #endif
     }()
 
-    @PublishedDefault(key: "isHeadTrackingEnabled", defaultValue: AppConfig.isHeadTrackingSupported)
+    @PublishedDefault(key: .isHeadTrackingEnabled, defaultValue: AppConfig.isHeadTrackingSupported)
```
Comment (Clstroud) [inline Vocable/AppConfig.swift]: Not sure that we need the key as an explicit label here, but we can refine separately

**Review by Clstroud — APPROVED** [review]
We paired on most of this so I had context. Good stuff! 🥇 💯

**Comment by CananArikan** [comment]
After enabling the listen mode from debug menu, first question/phrase is written to the screen and I am getting a response for it. After the first question/phrase, when I continue to ask questions, application is listening as you can see from the Xcode console. But, what I am asking/talking is not written to the screen as an expected output text. 

Tested:
iPad Pro - Test Device - iOS 15.3.1
iPhone 11 - Simulator - iOS 15.4

https://user-images.githubusercontent.com/100222793/165151294-e0ecba33-d638-4f00-9f88-f76983b55583.mp4

**Comment by jsmorgan42** [comment]
@CananArikan Fix has been pushed up!

**Comment by CananArikan** [comment]
@jsmorgan42 works perfect! Thanks!

**Review by Clstroud — APPROVED** [review]

**Pull request merged**